core/modules/layout/layout.admin.inc

```diff
@@ -1051,12 +1044,50 @@ function layout_block_add_page(Layout $layout, $renderer_name, $region_name, $bl
             'data-dialog-options' => json_encode(array('dialogClass' => 'layout-dialog')),
           )
         )
-      ),
-      '#description' => isset($block_info['description']) ? filter_xss($block_info['description']) : NULL,
-      '#wrapper_attributes' => array('class' => array('layout-block-add-row'))
+      );
+      $requirements_class = 'has-required-contexts';
+    }
+    else {
+      // Remove the main page content block since it's special.
+      if ($block_info['info'] == 'Main page content') {
+        continue;
+      }
+
+      $link = $block_info['info'];
+      $requirements_class = 'missing-required-contexts';
+
+      if (count($required_contexts) == 1) {
+        $required_context = reset($required_contexts);
+        $context_info = layout_get_context_info($required_context);
+        // Grab the first value in the menu paths, to use as an example.
+        $example_path = $context_info['menu paths'][0];
```

Comment:
This is addressing @quicksketch's comment from https://github.com/backdrop/backdrop/pull/1854#discussion_r114206079

The contexts returned now are in this form (as returned by `layout_get_context_info()`):

```
node/%node
taxonomy/term/%term
```

...whereas @jenlampton had previously hard-coded them to:

```
node/%
user/%
taxonomy/term/%
```